server/src/main/java/org/opensearch/action/search/SearchContextIdForNode.java

```diff
@@ -45,12 +45,12 @@
  *
  * @opensearch.internal
  */
-final class SearchContextIdForNode implements Writeable {
+public final class SearchContextIdForNode implements Writeable {
```

Comment:
Repeat comment: Why is this made public?

Reply:
It is used here - https://github.com/opensearch-project/OpenSearch/pull/3949/files#diff-2bca0e4d486c23e0913a6dc6e5481350180239b2a176cda1544ddb3c51b565c0R1440